### DISCUSSION TITLE
How do I change the bind address of a hono app (bun)?

### DISCUSSION THREAD

**Discussion by sinjs** [honojs/hono#1609] [Q&A] [answered]
Let's say I have the sample app:

```ts
// index.ts
const app = new Hono();

app.get("/", (ctx) => "Hello, world!");

export default app;
```

In the console it shows this:
```
$ bun run index.ts
Started server http://localhost:3000
```

How would I go about changing the bind address to something like `0.0.0.0` or `hostname.local`, i know how to change the port using `export default { port: 2000, ... }`, but no where in the documentation it mentions bind address.

**Comment by nakasyou** [comment, marked as answer]
The method is written in [Bun's document](https://bun.sh/docs/api/http#bun-serve).

```ts
export default {
  fetch: app.fetch,
  hostname: "mydomain.com",
}
```
But, It is always displayed as `localhost:xxxx`.
